docs/Training-On-Your-Data.md

```diff
@@ -369,9 +369,50 @@ This will spin up a TPU VM instance and install Levanter on it. You can then run
 gcloud compute tpus tpu-vm ssh my-tpu   --zone us-east1-d --worker=all --command="WANDB_API_KEY=... levanter/infra/launch.sh python levanter/src/levanter/main/train_lm.py --config_path gs://path/to/config.yaml"
 ```
 
-### GPU
 
-TODO, but you can mostly follow the guide for [TPU](#tpu) above.
+### GPU Training
+The [GPU Setup Guide](Getting-Started-GPU.md) covers how to do your first training runs with a model on GPU. Our [GPU Docker Development Guide](dev/GPU-Docker-Dev.md) explains how to setup a docker container for Levanter development, and our [Getting Started Training Guide](Getting-Started-Training.md)
+provides a detailed explanation of different training arguments and configurations.
+
+### Multi-Node GPU Training
+For multi-gpu training, you need to additionally have [nvidia-fabricmanager](https://docs.nvidia.com/datacenter/tesla/pdf/fabric-manager-user-guide.pdf) installed on each of your nodes.
+
+```
+sudo apt-get install cuda-drivers-fabricmanager
+sudo systemctl start nvidia-fabricmanager
+```
+
+If you are using a docker container to train your model, your docker run command should look similar to this
+
+```
+sudo docker run -it --network=host -v ~/src/levanter/cache:/cache -v /home/user/levanter:/levanter --gpus=all --shm-size=16g  ghcr.io/nvidia/jax:levanter
+```
+The main difference between the command here and the one found in the [GPU Docker Development Guide](dev/GPU-Docker-Dev.md) is the `--network=host` argument. This tells the docker container to use the host machine's network instead of the default docker `bridge` network. The `bridge` network is used for communication between docker containers that are on the same host machine, while the `host` network settings is better for container communication across different hosts. Since we are using multiple nodes, we want to use the `host` network setting. Please see the docker's [host](https://docs.docker.com/network/network-tutorial-host/) and [bridge](https://docs.docker.com/network/network-tutorial-standalone/) network documentation for more information.
```

Comment:
I don't think this is quite right. host is just the easiest way to do multinode, but there's overlay networks and stuff. I'd just say host networks are the easiest and are probably sufficient for this purpose